types/minio/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for minio 4.0
+// Type definitions for minio 5.0
 // Project: https://github.com/minio/minio-js#readme
 // Definitions by: Barin Britva <https://github.com/barinbritva>
```

Comment:
Need to add author :)

Reply:
I believe in this case I'll receive messages from definitely typed bot asking me to review other PRs, and I don't feel like I'm entitled to do that